doc/config.yml

```diff
@@ -43,6 +43,7 @@ Content:
             - python/CSVDiff.md
             - python/mms.md
             - python/MooseDocs/**
+            - ~python/MooseDocs/extensions/civet.md
```

Comment:
Is the `~` intentional here?

Reply:
The `~` is the `config.yml` syntax for "I don't want you to include the following path...."!

Reply:
Together, this means that I want to include everything underneath `python/MooseDocs/` *minus* `python/MooseDocs/extensions/civet.md`.